src/main/java/randoop/generation/AbstractGenerator.java

```diff
@@ -91,6 +91,13 @@ private long elapsedTime() {
    */
   protected final List<TypedOperation> operations;
 
+  /**
+   * A copy of {@link AbstractGenerator#operations} that contains all operations used to generate
```

Comment:
Is this really "all operations"?  Detective uses other operations to generate sequences.